### PR TITLE
Fixed the blocked clock when using the largest font and display size

### PR DESCRIPTION
**Pull request by yuhuitech**
Dear developer:

Hello! I am the creator of issue #257, and I have fixed the blocked clock when using the largest font and display size. I would appreciate it if you could kindly revise my code and leave me some advice. Thank you so much for your precious time!

Here is the screenshot after my changes:

copy copy 

Thanks again! Have a nice day! :)

### CODE REVIEW

**Comment by adrcotfas** [comment]
@yuhuitech Thank you so much for the contribution! I'll review the changes as soon as I can.

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -49,31 +49,29 @@
...
-            android:contentDescription="@string/tutorial_title" />
+            app:layout_constraintVertical_bias="0.5" />
 
         <TextView
             android:id="@+id/timeLabel"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:ellipsize="none"
+            style="@style/TimerStyle"
+            android:layout_width="match_parent"
```
[the remark this answers is not in the document]
Comment (yuhuitech) [reply app/src/main/res/layout/activity_main.xml:66]: OK, I will check this right now. :)

[the remark this answers is not in the document]
Comment (yuhuitech) [reply app/src/main/res/layout/activity_main.xml:66]: Hello! Sorry for that. After some tests, I guess I fixed this problem by setting the height of the clock form `match_parent` to 30% of the parent height. And I did some tests on Pixel 2 with API 25 and 29. They both seem fine to me. Here are screenshots after my latest modifications:

Pixel 2 with API 25:

copy
copy

and Pixel 2 with API of 29:

copy
copy

Thank you again! :)

[the remark this answers is not in the document]
Comment (adrcotfas) [reply app/src/main/res/layout/activity_main.xml:66]: Thank you so much for your contribution!

**Pull request merged**